connector_prestashop/models/product_template/importer.py

```diff
@@ -698,7 +702,10 @@ def import_images(self, binding):
             images = [images]
         for image in images:
             if image.get("id"):
-                delayable = self.env["prestashop.product.image"].with_delay(priority=10)
+                delayable = self.env["prestashop.product.image"].with_delay(
+                    priority=10,
+                    identity_key=identity_exact,
```

Comment:
What does it do?

Reply:
@maciej-wichowski it makes sure:
`When used, this identity key will have the effect that when a job should be created and a pending job with the exact same recordset and arguments, the second will not be created.`